### ISSUE TITLE
Beavers initial promotion - multiple issues in one issue

### ISSUE THREAD

**Issue by ppedziwiatr** [warp-contracts/cyberpunk-beavers#32]
1. Proste, degenskie posty shillujące bobry (max co 1-2 dni) grafika i 2-3 zdania. Cos w stylu:
https://x.com/berachain, https://x.com/CryptoHayes

2. Założenie kont na Galxe, https://app.layer3.xyz/, https://guild.xyz/explorer

3. Partnerships - dystrybucja CRED, albo inne tokeny z ecosystemu Arweave

4. (TBD with tech team) mozliwość wykopania coinów z innych chainów (w pierwszej kolejności z ETH)

**Comment by stasiakmatt** [comment]
Short description:

Posts on X: Since 2nd July, I’ve been posting one post daily on X. The most important ones are the gameplay posts. The first one from 2nd July got 16k views and was reposted by Sam, the second gameplay post got over 2k views, so the game is the most interesting. There were also initial updates about adding the token via ArConnect, Helix, rewards for the first gameplay, a repost from Redstone at EthCC, and information about the tournaments. The plan includes follow-ups to change the name in Helix, reminders about adding the token, updates about the gameplay, etc.

Info on Discord and about the games: Managed, with gameplay and daily questions answered, but I still need to organize general informations about the gameplay and the game, working on that.

Account setup and role preparation: After meeting with Jawor, the plan was established to use Guild + Zealy, with Galxe in the third place (there is a deposit here). Not using https://app.layer3.xyz/.
The Guild with roles was set up, I posted it for feedback, and then yesterday I posted the first tweet online.
Now I want to post on Discord about creating maps and possibly engage Heyo, who is already on the Guild and wants to make maps, to get started.

Partnerships:

wAR from Forward: According to David, CRED and AO are difficult to convey, but we are in talks with Forward Research about wrapped AR. An initial email with a proposal for $1000 wAR for distribution was sent to them, got response that it is in their operations and accounting, assuming for procedings and decisions.
Jonathan from ArDrive and EXP: I have a call with Jonathan to finalize details, already scheduled for first possible term next week; they prefer EXP tokens, but the amount has not been determined, also test IO token is possible, as it was mentioned.
Trunk: Ringo is on a long vacation, he is the decision-maker, I sent 2 messages, and we are waiting for his return from vacation. Simonfriend contacted, yet is not involved and he is not a decisionmaker.

Will create 4 tasks to monitore them and keep updates, as 1 task is to big to move between Doing and Done, as parts are in progress at the same time.
